src/AddressBookUI/ABPersonViewController.cs

```diff
@@ -108,21 +110,21 @@ partial class ABPersonViewController {
 		public void SetHighlightedItemForProperty (ABPersonProperty property, int? identifier)
 		{
 			SetHighlightedItemForProperty (
-					ABPersonPropertyId.ToId (property),
-					identifier.HasValue ? identifier.Value : ABRecord.InvalidPropertyId);
+					property,
```

Comment:
It looks like this is a difference in behavior: this will run into infinite recursion, because you removed the call to ABPersonPropertyId.ToId. That line should be reverted:

```suggestion
					ABPersonPropertyId.ToId (property),
```

Reply:
@rolfbjarne  Ah good catch! I didn't realize I was calling the same method... The reason I made this change is that the SetHighlightedItemForProperty (int property, int identifier) is Obsolete and suggests that I use this one - which in this case does not make sense. Should I in fact continue to use the obsoleted one?

Reply:
The `SetHighlightedItemForProperty (int, int)` overload is both internal and obsolete - which makes the obsolete redundant because nobody will see it, and you can just remove the obsolete attribute:

https://github.com/xamarin/xamarin-macios/blob/5cc92ac564ece03efb79c9b271beafc8faf905c0/src/addressbookui.cs#L164-L169

The comment is also out of date, so it can be removed too.

Reply:
@mandel-macaque This should answer your question below, however, I can add this reason in the PR description